src/main/resources/root.jsp

```diff
@@ -17,6 +17,23 @@
 
     <internal:gwtGenerateDictionary/>
     <internal:gwtInit/>
+
+    <script type="text/javascript">
```

Comment:
why moved the code here ?

Reply:
because GWT needs it to get initialised properly